### PR TITLE
add installation guidelines to the README file 

### LINKED ISSUE #197: Add package installation guidelines to README.md
Hey, 
I'd like to add the package's installation guidelines to the README file. 
This would make it easier for the users to install the package in their projects by simply copying and pasting the command to their terminals. 

Could this issue be assigned to me? 

Let me know what you think of the above.

### PR DESCRIPTION
**Pull request by morelmiles**
Fixes #197

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,20 @@ The part in the middle is the interesting bit.  It's called the Claims and conta
 
 This library supports the parsing and verification as well as the generation and signing of JWTs.  Current supported signing algorithms are HMAC SHA, RSA, RSA-PSS, and ECDSA, though hooks are present for adding your own.
 
+## Installation Guidelines 
+
+1. To install the jwt package, you first need [Go](https://go.dev/doc/install) installed, then you can use the below Go command to install jwt.
+
+```sh
+$ go get -u github.com/golang-jwt/jwt
```
Comment (qqiao) [inline README.md:44]: I think you acutally want github.com/golang-jwt/jwt/v4 here

File: README.md [hunk 2]
```diff
@@ -36,6 +36,20 @@ The part in the middle is the interesting bit.  It's called the Claims and conta
...
+```
+
+2. Import it in your code:
+
+```go
+import "github.com/golang-jwt/jwt"
```
Comment (qqiao) [inline README.md:50]: I think you actually want github.com/golang-jwt/jwt/v4 here

Comment (morelmiles) [reply README.md]: Hey @qqiao, i've sent the fix. 

Cheers,

**Review by mfridman — DISMISSED** [review]
lgtm.

**Review by oxisto — APPROVED** [review]

**Pull request merged**